.github/workflows/shallow_checkpatch.yml

```diff
@@ -51,3 +48,15 @@ jobs:
       - name: checkpatch --strict
         run: .github/workflows/checkpatch_list.sh ${CHK_CMD_OPTS}
                  --strict < PR_SHAs.txt
+  yamllint:
```

Comment:
Have you considered moving this to a new `yamllint.yml` file? With hindsight I think using one big file for everything was a Github Action beginner mistake.

Reply:
I have nothing against it. i just figured it would make more sense to patch sanity check tools into the same workflow "yamllint, checkpatch, pep8, etc.." thoughts? I agree things like build tools or actual builds should be separate files

Reply:
> I just figured it would make more sense to patch sanity check tools into the same workflow "yamllint, checkpatch, pep8, etc.." thoughts?

Why not, in that case maybe change the titles a bit because right now this is called "checkpatch" at every level: workflow, job and what not. Warning: github actions "headings" look slightly different in merged vs unmerged PRs, have a look at one example of each.